components/net/fetch/methods.rs

```diff
@@ -40,6 +40,10 @@ pub fn fetch_async(request: Request, listener: Box<AsyncFetchListener + Send>) {
 
 /// [Fetch](https://fetch.spec.whatwg.org#concept-fetch)
 pub fn fetch(request: Rc<Request>) -> Response {
+    fetch_with_cors_cache(request, &mut BasicCORSCache::new())
+}
+
+pub fn fetch_with_cors_cache<C: CORSCache>(request: Rc<Request>, cache: &mut C) -> Response {
```

Comment:
`CORSCache` is a trait, so you'll get the following error.
```
error: the trait bound `<trait>: std::marker::Sized` is not satisfied
```
I couldn't think of another way around it